src/GenomeCopyNumber.cpp

```diff
@@ -3558,35 +3661,35 @@ void GenomeCopyNumber::addBAFinfo(SNPinGenome & snpingenome) {
 
         chrCopyNumber_[index].addBAFinfo(snpingenome,indexSNP);
 
-		//int length = chrCopyNumber_[index].getLength();
+		int length = chrCopyNumber_[index].getLength();
 
 		//create a vector with BAF
-//		chrCopyNumber_[index].createBAF(NA);
-//        int totalSNPnumber = snpingenome.SNP_atChr(indexSNP).getSize() ;
-//		int SNPcount = 0;
-//		float currentBAF = snpingenome.SNP_atChr(indexSNP).getValueAt(SNPcount);
-//		int getSNPpos = snpingenome.SNP_atChr(indexSNP).getPositionAt(SNPcount);
-//        float minBAF;
-//		for (int i = 0; i<length; i++) {
-//		    int left = chrCopyNumber_[index].getCoordinateAtBin(i);
-//		    int right = chrCopyNumber_[index].getEndAtBin(i);
-//		    if (getSNPpos>=left && getSNPpos <=right) {
-//		        minBAF = chrCopyNumber_[index].getBAFat(i);
-//                if (minBAF==NA) {
-//                    chrCopyNumber_[index].setBAFat(i,currentBAF);
-//                } else {
-//                    chrCopyNumber_[index].setBAFat(i,min(minBAF,currentBAF));
-//                }
-//		    } else if (getSNPpos<left) {
-//                if (SNPcount < totalSNPnumber) {
-//                    SNPcount++;
-//                    getSNPpos = snpingenome.SNP_atChr(indexSNP).getPositionAt(SNPcount);
-//                    currentBAF = snpingenome.SNP_atChr(indexSNP).getValueAt(SNPcount);
-//                    i=max(-1,i-windowSize_/step_-1);
-//                    //cout << SNPcount << " out of "<< totalSNPnumber<<"\n";
-//                }
-//            }
-//		}
+		chrCopyNumber_[index].createBAF(NA);
```

Comment:
Can you explain this change?

Reply:
this is strange. I don't remember touching this block...

Reply:
I think I removed the comment in order to generate output BAF file.